pkg/apis/kubeflow/v2beta1/types.go

```diff
@@ -131,6 +139,17 @@ type RunPolicy struct {
 	// Defaults to false.
 	// +kubebuilder:default:=false
 	Suspend *bool `json:"suspend,omitempty"`
+
+	// ManagedBy is used to indicate the controller or entity that manages a MPIJob.
+	// The value must be either an empty, 'kubeflow.org/mpi-operator' or
```

Comment:
```suggestion
	// The value must be either empty, 'kubeflow.org/mpi-operator' or
```

Reply:
@mszadkow Could you address this comment in a separate PR?

Reply:
Sure, will be addressed asap